migtests/scripts/run-schema-migration.sh

```diff
@@ -118,7 +118,7 @@ main() {
         mv "${EXPORT_DIR}/schema/failed.sql" "${EXPORT_DIR}/schema/failed.sql.bak"
         #replace_files
         replace_files "${TEST_DIR}/replacement_dir" "${EXPORT_DIR}/schema"
-        import_schema
+        import_schema --start-clean t
```

Comment:
why start-clean required here?
Please add a small comment also.

Reply:
It's required since we are running import schema here for the second time after fixing the issues. Adding the comment.